apps/babanews/src/components/babanews/fullwidth-banner/fullwidth-banner.tsx

```diff
@@ -0,0 +1,25 @@
+import {css, Theme, useTheme} from '@mui/material'
+import {styled} from '@mui/material'
+import {BuilderBreakBlockProps, useWebsiteBuilder} from '@wepublish/website'
+import {useMemo} from 'react'
+
+const Banner = styled('div')`
+  aspect-ratio: 2.7/1;
+  width: 100%;
+  background-color: ${({theme}) => theme.palette.info.main};
+`
+
+const imageStyles = (theme: Theme) => css`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
```

Comment:
Check this on safari/iOS might cause issues. We have the same bug for our teasers rn

Reply:
checked on safari, no issues. I don't have an ios device